VariationalBayes/ExponentialFamilies.py

```diff
@@ -117,6 +117,48 @@ def get_e_log_dirichlet(alpha):
     return sp.special.digamma(alpha) - digamma_sum
 
 
+def get_e_fun_logitnormal(lognorm_means, lognorm_infos, \
+                        gh_loc, gh_weights, fun):
+    # compute E(fun(X)) where X is a vector of lognormals defined by parameters
+    # lognorm_means and lognorm_infos
+
+    change_of_vars = np.sqrt(2) * gh_loc[:, None] * \
```

Comment:
This will not work with arbitrarily shaped means and infos.  (Please add a unit test to confirm this, and also to confirm that your final solution does the trick.)

For an example of a generic way to do this, maybe check out the GLMM code:
https://github.com/rgiordan/LinearResponseVariationalBayes.py/blob/203c2df2e3545b4ddefba81c4f5e8c0a7cb98db2/VariationalBayes/Modeling.py#L35